### PR TITLE
Improve READMEs

### PR DESCRIPTION
**Pull request by roj1512**

### CODE REVIEW

File: vercel-bot/README.md [hunk 1]
```diff
@@ -1,7 +1,8 @@
-## vercel_grammy_bot
+# [grammY](https://grammy.dev) on [Vercel](https://vercel.dev)
 
-You can deploy your bot to Vercel as a serverless function. Here is a bot running there:
+You can deploy your bots to Vercel as a serverless function.
 
-https://t.me/vercel_grammy_bot
+> Try it out: [@vercel_example_bot](https://t.me/vercel_example_bot)
```
Comment (Loskir) [inline vercel-bot/README.md:5]: hmm, why did vercel_grammy_bot change to vercel_example_bot?

Comment (roj1512) [reply vercel-bot/README.md:5]: mm

Comment (roj1512) [reply vercel-bot/README.md:5]: мой бад

**Comment by Loskir** [comment]
I think we should add some steps to follow in the vercel example

**Comment by roj1512** [comment]
That's for another PR.

**Review by Loskir — APPROVED** [review]
Okay 👍

**Pull request merged**